### DISCUSSION TITLE
Using GLB model for ios quickview and android scene viwer does not load the model at 100% scale in scene viewer, however it does in quickview.

### DISCUSSION THREAD

**Discussion by Carlos19979** [google/model-viewer#3960] [Q&A]
### Description
Using GLB model for ios quickview and android scene viwer does not load the model at 100% scale in scene viewer, however it does in quickview, I'm not using fixed ar scale.

#### Live Demo
http://metaplainarhomes.s3-website-eu-west-1.amazonaws.com/dashboard/promotion/viviendas/real

### Version

- model-viewer: v2.1.0

### Browser Affected

- [x ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x ] Safari

### OS

- [x ] Android
- [x ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [x ] SceneViewer
- [x ] QuickLook

**Comment by elalish** [comment]
This is by design; Scene Viewer scales down huge assets to help them be visible and that behavior is not controllable with their API. Please use WebXR mode instead, which doesn't have this behavior.
